libqaul/src/lib.rs

```diff
@@ -2,19 +2,19 @@
 
 mod auth;
 mod crypto;
-mod users;
+pub mod users;
 mod storage;
 
 // This module defines the libqaul service API
 mod api;
 pub use api::*;
 
-use identity::Identity;
+pub use identity::Identity;
 use std::{
     collections::BTreeMap,
     sync::{Arc, Mutex},
 };
-use users::User;
+use users::{User};
```

Comment:
I guess this used to be more than one import? :D

Reply:
Ah yeah, guess I forgot to run `rustfmt` for that last commit.